discovery-provider/plugins/verified-uploads/src/handlers/tracks.js

```diff
@@ -1,79 +1,62 @@
-import { dp_db } from "../db.js";
-import { slack } from "../slack.js";
-import dotenv from "dotenv";
+import { dp_db } from '../db.js'
+import { slack } from '../slack.js'
+import dotenv from 'dotenv'
 
-dotenv.config();
-const { TRACKS_SLACK_CHANNEL } = process.env;
+dotenv.config()
+const { TRACKS_SLACK_CHANNEL } = process.env
 
 export const isOldUpload = (uploadDate) => {
-  let uploadedDate = new Date(uploadDate).getTime();
-  let oneWeekAgo = new Date();
-  oneWeekAgo.setDate(oneWeekAgo.getDate() - 7);
-  oneWeekAgo = oneWeekAgo.getTime();
-  return oneWeekAgo > uploadedDate;
-};
+  let uploadedDate = new Date(uploadDate).getTime()
+  let oneWeekAgo = new Date()
+  oneWeekAgo.setDate(oneWeekAgo.getDate() - 7)
+  oneWeekAgo = oneWeekAgo.getTime()
+  return oneWeekAgo > uploadedDate
+}
 
-export default async ({ track_id }) => {
-  const trackId = track_id;
-  const results = await dp_db("tracks")
-    .innerJoin("users", "tracks.owner_id", "=", "users.user_id")
-    .innerJoin("track_routes", "tracks.track_id", "=", "track_routes.track_id")
+export default async ({ track_id, updated_at, created_at }) => {
+  const isUpload = updated_at === created_at
+  if (!isUpload) return
+
+  const trackId = track_id
+  const results = await dp_db('tracks')
+    .innerJoin('users', 'tracks.owner_id', '=', 'users.user_id')
+    .innerJoin('track_routes', 'tracks.track_id', '=', 'track_routes.track_id')
```

Comment:
note this won't necessarily get the most up to date track_route slug but probably ok for this use case

Reply:
good point, adding is_current seems like the right call? we still have that on slugs right?